File: proxy/quickstart/create-session/create-session.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require './vendor/autoload.php';
use Twilio\Rest\Client;
// Get your Account SID and Auth Token from https://twilio.com/console
// To set up environmental variables, see http://twil.io/secure
$sid = getenv('TWILIO_ACCOUNT_SID');
$token = getenv('TWILIO_AUTH_TOKEN');
// Initialize the client
$client = new Client($sid, $token);
$session = $client
    ->proxy
    ->services("KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
    ->sessions->create(
        array(
            'uniqueName' => "MyFirstSession"
        )
    );

echo $session->uniqueName;

File: proxy/create-session/create-session.5.x.cs
using System;
using Twilio;
using Twilio.Rest.Proxy.V1.Service;


class Example
{
    static void Main(string[] args)
    {
      // Find your Account Sid, Auth Token and Proxy Service sid at twilio.com/console
      // To set up environmental variables, see http://twil.io/secure
      const string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
      const string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
      const string proxyServiceSid = "KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
      TwilioClient.Init(accountSid, authToken);

      var session = SessionResource.Create(proxyServiceSid, "MyFirstSession");

      Console.WriteLine(session.Sid);
    }
}
